examples/test/test_single_agent.py

```diff
@@ -36,9 +37,12 @@ def test_misalignment_single_agent(model):
     examples.misalignment.single_agent.main(model=model)
 
 
-@parametrize
+@pytest.mark.parametrize('model', [
+    ModelType.STUB,
+])
 def test_evaluation_single_agent(model):
-    examples.evaluation.single_agent.main(model=model)
+    with patch('time.sleep', return_value=None):
```

Comment:
Where is the `time.sleep` in `examples.evaluation.single_agent`?

Reply:
Thanks for spotting. I did not find it, but assumed that it is there somewhere :) Since this example is very slow.

Reply:
Reverted